index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <h1 id="hello">Hello World, we are software developers!</h1>
+    <h1 id="title">Survey Form</h1>
+    <p id="description"><b>This a very important survey. Please participate.</b></p>
+    <form action="" id="survey-form">
+     
+      <label for="name" id="name-label"><b>Name: </b></label>
+      <input type="text" id="name" placeholder="Enter your name"> <br> <br>
+
+      <label for="emial" id="email-label"><b>Email: </b></label>
+      <input type="email" id="email" placeholder="Enter your email address">  <br> <br>
+
+      <label for="number" id="number-label"><b>Number: </b></label>
```

Comment:
- [ ] Once again please make sure this comment is phrased politely as a suggestion as well.

Reply:
- [Optional] Good work catching that typo. But i would suggest that it be changed to number instead of number-label as this is an unnecessary repetition 👍🏾